### PR TITLE
[SofaKernel] Brainless Warnings cleaning

### PR DESCRIPTION
**Pull request by marques-bruno**
______________________________________________________

This PR: 
- [ ] builds with SUCCESS for all platforms on the CI.
- [ ] does not generate new warnings.
- [ ] does not generate new unit test failures.
- [ ] does not generate new scene test failures.
- [ ] does not break API compatibility.
- [ ] is more than 1 week old (or has fast-merge label).

**Reviewers will merge only if all these checks are true.**

### CODE REVIEW

**Comment by guparan** [comment]
Hey @marques-bruno thanks for this PR. Could you please resolve the conflicts so that CI can try to build it? :smile:

File: SofaKernel/framework/sofa/core/objectmodel/BaseContext.cpp [hunk 1]
```diff
@@ -107,7 +107,7 @@ BaseContext* BaseContext::getRootContext() const
 void* BaseContext::getObject(const ClassInfo& /*class_info*/, SearchDirection /*dir*/) const
 {
     serr << "calling unimplemented getObject method" << sendl;
```
Comment (epernod) [inline SofaKernel/framework/sofa/core/objectmodel/BaseContext.cpp]: maybe good time to remove the serr ?

File: SofaKernel/framework/sofa/core/objectmodel/BaseContext.cpp [hunk 2]
```diff
@@ -116,7 +116,7 @@ void* BaseContext::getObject(const ClassInfo& /*class_info*/, SearchDirection /*
 void* BaseContext::getObject(const ClassInfo& /*class_info*/, const TagSet& /*tags*/, SearchDirection /*dir*/) const
 {
     serr << "calling unimplemented getObject method" << sendl;
```
Comment (epernod) [inline SofaKernel/framework/sofa/core/objectmodel/BaseContext.cpp]: same serr

**Comment by marques-bruno** [comment]
[with-regression-tests][with-scene-tests]

**Comment by marques-bruno** [comment]
[ci-build][with-scene-tests][with-regression-tests]

**Pull request merged**